### ISSUE TITLE
Keyboard navigation doesn't work in ranking example

### ISSUE THREAD

**Issue by ST-1580** [surveyjs/survey-library#4337]
Steps to reproduce:
1) Open [the example](https://surveyjstest.azurewebsites.net/Examples/Library?id=questiontype-ranking&platform=Knockoutjs)
2) Select Knockout framework
3) Go to the plunker example
4) Click to `Battery life` item
5) When click `tab` button nothing happens

**Comment by dmitrykurmanov** [comment]
It seems that it has already fixed.
